app/models/arclight/parents.rb

```diff
@@ -9,7 +9,7 @@ class Parents
     def initialize(ids:, labels:, eadid:)
       @ids = ids
       @labels = labels
-      @eadid = eadid
+      @eadid = eadid.tr('.', '-')
```

Comment:
i've seen this logic in a few places. we could add a `normalize_ead` function or something for it. 

see https://github.com/sul-dlss/arclight/blob/ed73b69696dd5f33ae8835b6fc9f802a9d1ea10f/lib/arclight/solr_ead_indexer_ext.rb#L29

https://github.com/sul-dlss/arclight/blob/3d911d0220ef7a00b082d45ec0cd242a650446de/lib/arclight/custom_document.rb#L45

https://github.com/sul-dlss/arclight/blob/3d911d0220ef7a00b082d45ec0cd242a650446de/lib/arclight/custom_component.rb#L27